scopes/envs/envs/environment.ts

```diff
@@ -1,3 +1,17 @@
+import { Linter } from '@teambit/linter';
+import { Tester } from '@teambit/tester';
+import { Bundler, BundlerContext, DevServer, DevServerContext } from '@teambit/bundler';
+import { BuildTask } from '@teambit/builder';
```

Comment:
so are we good with having all these as dependencies of the Env aspect?
Don't we get circular dependencies? envs -> builder -> envs.

In general, I like the approach taken in this file, it'd be must easier to reason about the env interface.